bot/cogs/animals.py

```diff
@@ -0,0 +1,127 @@
+"""Commands for the animal module."""
+import discord
+
+from discord.ext.commands import Cog, bot, command, context
+
+
+class AnimalCog(Cog, name='Animals'):
+    """Commands for the animal finder."""
+
+    def __init__(self, bot: bot.Bot):
+        self.bot = bot
```

Comment:
`bot` is used three different ways here, if we correct the import, then we can use `bot: Bot` instead. Or better yet, we can import `Friendo` from `bot.bot` and do `def __init__(self, bot: Friendo):`